documentation/smg/10_build_release_process.md

```diff
@@ -55,52 +70,37 @@ Disadvantages:
 
 - slower; have to wait for full build and test pipeline to execute
 
-### Releases
 
-The Jenkins build artifacts are are not accessible outside of STFC. End users will be directed to GitHub to access packaged releases.
 
-A 'deploy' pipeline will be created ([Horace #73](https://github.com/pace-neutrons/Horace/issues/73)). 
+### Hot-fix Releases
 
-This will:
+If a particular fix is required quickly on a target system so that facility users to not lose critical machine time the full build and test process can be bypassed. 
 
-- tag the `master` branch with the release number (`Rm_n_o`)
-- copy the built artifact to GitHub
+Quick modifications to the software, with only partial testing, carry the risks of breaking other parts of the software, so this is an exceptional use-case. Non-time critical issues should be resolved through the standard Patch Release process.
 
-The release tag will be an anchor for any subsequently needed release branch and patch branches.
+A "hot-fix" pipeline has been created in ANVIL that does not run tests and only builds and packages Herbert or Horace. This will complete is around 5 minutes as it is the test execution step of the standard PR pipeline that take in excess of an hour to run.
```

Comment:
```suggestion
A "hot-fix" pipeline has been created in ANVIL that does not run tests and only builds and packages Herbert or Horace. This will complete in around 5 minutes as it is the test execution step of the standard PR pipeline that take in excess of an hour to run.
```

Reply:
We don't have this pipeline yet, we should open an issue for this.

Might also be worth splitting this line up into a few lines